app/javascript/components/util/InfoTooltip.js

```diff
@@ -345,7 +345,8 @@ const TOOLTIP_TEXT = {
   analyticsAgeTip: (
     <div>
       Current Age is calculated as:
-      <div>Current Date - Date of Birth</div>
+      <br />
```

Comment:
We shouldn't be using line breaks like this. use a `<div/>` or other styling instead.

Reply:
I disagree, we shouldn't add extra tags just for the sake of it.  Besides, they're used all over the place in this component.  I did however clean up this file

Reply:
We'll agree to disagree on this one.